config/example/rank_colours.txt

```diff
@@ -0,0 +1,11 @@
+# Use this file to denote colours for ingame admin ranks, using the following format
```

Comment:
Not sure it makes sense to include Paradise's staff structure as default, considering some of these ranks don't appear in `config/example/admin_ranks.txt`